### DISCUSSION TITLE
question about normalization in CIGALE

### DISCUSSION THREAD

**Discussion by bwv1194** [mboquien/cigale#44] [Q&A]
Hi,

I have a best-fit stellar model of AGN host galaxy which is derived from SED fitting of pure-host flux obtained from optical image decomposition. Now when performing UV-MIR SED decomposition, is it possible to fix the stellar model to that obtained from image decomposition so that the stellar mass derived from the two methods are the same? I tried to fix the stellar parameters to the best-fit values but do not know how to get the correct normalization.

**Comment by mboquien** [comment]
You probably want to set the `normalize` parameter to `false` and then give the exact scaling (it depends on the exact module, it can be `sfr_0` or `sfr_A` typically), which should be given by `best.sfh.integrated` I think.

**Reply by bwv1194** [reply]
I tried this method but sfr_A is treated as a free parameter in the fitting, and the fitting result is different from my input single value (depends on the allowed value for the AGN fraction). I guess what I need is to find a way to fix this parameter.

**Reply by mboquien** [reply]
If you put a single value, then the run has to be limited to that one object. Another option is to give all the values and fit sfr_A as a parameter. To make sure that only the correct value is taken into account, in your input flux file, add a column for sfr_A and give en extremely small value for sfr_A_err.
